### PR TITLE
Footer update1

### PR DESCRIPTION
**Pull request by MatthewNeie**
modify footer to include roles, bold names, add GitHub repo link, and spacing changes

### CODE REVIEW

**Comment by MatthewNeie** [comment]
I think we should have our first and last names in the team section. Also, please add your roles to README so I can update when I have info from everyone.

**Review by belunatic — APPROVED** [review]

File: expense-splitter/src/components/layout/Footer.jsx
Comment (singhalex) [inline expense-splitter/src/components/layout/Footer.jsx:1]: You can remove the React import at line 1. It is no longer needed with newer version of React.

Also, can you apply the prettier formatting for consistency and readability? If you don't already, I would recommend installing the prettier extension and enabling auto format on save.

Comment (ccespedes) [reply expense-splitter/src/components/layout/Footer.jsx:1]: I updated the readme, I suggest you post your request in discord for others to follow.

**Review by ccespedes — COMMENTED** [review]
I updated the readme, I suggest you post your request in discord for others to follow.

**Comment by belunatic** [comment]
added my credentials already

**Review by ccespedes — APPROVED** [review]
I think everyones creds are in there now so I'll merge.

**Pull request merged**